server/src/main/scala/za/co/absa/atum/server/api/controller/PartitioningController.scala

```diff
@@ -23,6 +23,6 @@ import zio.macros.accessible
 
 @accessible
 trait PartitioningController {
-  def createPartitioningIfNotExists(partitioning: PartitioningSubmitDTO): IO[ErrorResponse, AtumContextDTO]
+  def createPartitioningIfNotExists(partitioningSubmitDTO: PartitioningSubmitDTO): IO[ErrorResponse, AtumContextDTO]
```

Comment:
Because I wanted to distinguish between the partitioning parameter for this method and for `partitioningMeasure's` and `partitioningAdditionalData` methods